x/oracle/client/cli/txUpgradeOracleProposal.go

```diff
@@ -0,0 +1,100 @@
+package cli
+
+import (
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	govcli "github.com/cosmos/cosmos-sdk/x/gov/client/cli"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	"github.com/medibloc/panacea-core/v2/x/oracle/types"
+	"github.com/spf13/cobra"
+)
+
+const (
+	FlagUpgradeUniqueID = "upgrade-unique-id"
+	FlagUpgradeHeight   = "upgrade-height"
+)
+
+func CmdUpgradeOracleProposal() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "oracle-upgrade",
```

Comment:
As done in before, let's add instructions for flags

```suggestion
		Use:   "oracle-upgrade (--upgrade-unique-id [uniqueID]) (--upgrade-height [height]) [flags]",
```